tests/metrics/run.sh

```diff
@@ -99,22 +99,25 @@ function run() {
 	inject_points=(
 		"github.com/pingcap/dm/syncer/changeTickerInterval=return(5)"
 		"github.com/pingcap/dm/syncer/noJobInQueueLog=return()"
-		"github.com/pingcap/dm/syncer/IgnoreSomeTypeEvent=return(\"HeartbeatEvent\")"
+		"github.com/pingcap/dm/syncer/OnlySomeTypeEventCanPass=return(\"WriteRowsEventV2\")"
 	)
 	# Since the following test needs to ensure that the dml queue is empty for a long time,
-	# it needs to ignore upstream heartbeat events to ensure that flushjobs are not triggered
+	# It is necessary to ensure that only such events `WriteRowsEventV2` are currently processed
 	export GO_FAILPOINTS="$(join_string \; ${inject_points[@]})"
 	# First set the ticker interval to 5s -> expect the execSQL interval to be greater than 5s
 	# At 5s, the first no job log will appear in the log
 	# At 6s, the ticker has already waited 1s and the ticker goes to 1/5th of the way
 	# At 6s, a dml job is added to job chan and the ticker is reset
 	# At 11s the ticker write the log of the second no job
 	# Check that the interval between the two ticker logs is > 5s
+
+	rm -rf $WORK_DIR/worker1/log/dm-worker.log # clean up the old log
 	run_dm_worker $WORK_DIR/worker1 $WORKER1_PORT $cur/conf/dm-worker1.toml
 	check_rpc_alive $cur/../bin/check_worker_online 127.0.0.1:$WORKER1_PORT
 
 	echo "sleep 6s"
 	sleep 6
```

Comment:
maybe this sleep is no longer need, because check_log_contain_with_retry will find a proper sleep time